e2e/content-services/directives/delete-directive.e2e.ts

```diff
@@ -0,0 +1,269 @@
+import { FolderModel } from '../../models/ACS/folder.model';
+
+describe('Delete Directive', () => {
+
+    this.alfrescoJsApi = new AlfrescoApi({
+        provider: 'ECM',
+        hostEcm: browser.params.testConfig.adf_acs.host
+    });
+
+    const loginPage = new LoginPage();
+    const contentServicesPage = new ContentServicesPage();
+    const paginationPage = new PaginationPage();
+    const contentListPage = contentServicesPage.getDocumentList();
+    const acsUser = new AcsUserModel();
+    const secondAcsUser = new AcsUserModel();
+    const uploadActions = new UploadActions(this.alfrescoJsApi);
+    const permissionActions = new PermissionActions(this.alfrescoJsApi);
+    let baseFolderUploaded;
+
+    const txtFileModel = new FileModel({
+        name: browser.params.resources.Files.ADF_DOCUMENTS.TXT.file_name,
+        location: browser.params.resources.Files.ADF_DOCUMENTS.TXT.file_location
+    });
+
+    const file0BytesModel = new FileModel({
+        name: browser.params.resources.Files.ADF_DOCUMENTS.TXT_0B.file_name,
+        location: browser.params.resources.Files.ADF_DOCUMENTS.TXT_0B.file_location
+    });
+
+    const pdfFileModel = new FileModel({
+        name: browser.params.resources.Files.ADF_DOCUMENTS.PDF.file_name,
+        location: browser.params.resources.Files.ADF_DOCUMENTS.PDF.file_location
+    });
+
+    const secondPdfFileModel = new FileModel({
+        name: browser.params.resources.Files.ADF_DOCUMENTS.PDF_B.file_name,
+        location: browser.params.resources.Files.ADF_DOCUMENTS.PDF_B.file_location
+    });
+
+    const pngFileModel = new FileModel({
+        name: browser.params.resources.Files.ADF_DOCUMENTS.PNG.file_name,
+        location: browser.params.resources.Files.ADF_DOCUMENTS.PNG.file_location
+    });
+
+    const secondPngFileModel = new FileModel({
+        name: browser.params.resources.Files.ADF_DOCUMENTS.PNG_B.file_name,
+        location: browser.params.resources.Files.ADF_DOCUMENTS.PNG_B.file_location
+    });
+
+    const folderInfo = new FolderModel({
+        name: 'myFolder',
+        location: browser.params.resources.Files.ADF_DOCUMENTS.TEXT_FOLDER.folder_location
+    });
+
+    const folderSecond = new FolderModel({
+        name: 'myrSecondFolder',
+        location: browser.params.resources.Files.ADF_DOCUMENTS.TEXT_FOLDER.folder_location
+    });
+
+    const baseFolder = new FolderModel({
+        name: 'BASE_FOLDER',
+        location: browser.params.resources.Files.ADF_DOCUMENTS.TEXT_FOLDER.folder_location
+    });
+
+    beforeAll(async () => {
+        await this.alfrescoJsApi.login(browser.params.testConfig.adf.adminEmail, browser.params.testConfig.adf.adminPassword);
+        await this.alfrescoJsApi.core.peopleApi.addPerson(acsUser);
+        await this.alfrescoJsApi.core.peopleApi.addPerson(secondAcsUser);
+        await this.alfrescoJsApi.login(acsUser.id, acsUser.password);
+    });
+
+    beforeEach(async () => {
```

Comment:
no need to create the baseFolder in beforeEach and delete it in afterEach.
the baseFolder is not changed in any way in the tests, so this should be moved in beforeAll and delete in afterAll for the suite

Reply:
The trick of the base folder is to clean up the environment after each test, so i with a single delete i'm able to delete all the content i've uploaded as the content are different in some cases. That's why i need to create it every time as in the end i'll delete just that one.

Reply:
ah yes, makes sense for this purpose